config/example.application.yml

```diff
@@ -102,6 +102,7 @@ defaults: &defaults
   graphhopper_url: "https://graphhopper.com/api/1/route"
   mapquest_directions_url: "https://open.mapquestapi.com/directions/v2/route"
   osrm_url: "https://router.project-osrm.org/route/v1/driving/"
```

Comment:
This line can be removed I think?

Reply:
Assuming MQ doesn't rise from the semi-dead yes

Reply:
I meant `osrm_url` actually - the MQ one does still work if you have an API key though I guess it may just be using the non-open routing now?

Reply:
I actually removed both and other lingering bits referring to MQ. Easy enough to resurrect in the unlikely case of them ever coming back.